### PR TITLE
Variables docs

### PR DESCRIPTION
**Pull request by billpalombi**
Needs examples, especially of accessing a variable in a config file.

### CODE REVIEW

File: docs/concepts/variables.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# Variables
+
+Variables enable you to store and reuse non-sensitive bits of data, often configuration information. Variables are dynamic-named, mutable string values, much like environment variables. Variables are scoped to a Prefect Server instance or a single workspace in Prefect Cloud.
+
+Variables can be created or modified at any time, but are intended for values with infrequent writes and frequent reads. Variable values may be cached for quicker retrival.
+
+While variable values are most commonly loaded during flow runtime, they can be loaded in other contexts, at any time, such that they can be used to pass configuration information to Prefect configuration files, such as project steps.
+
+!!! warning "Variables are not Encrypted"
+    Using variables to store sensitive information, such as credentials, is not reccomended. Instead, use [Secret blocks](https://docs.prefect.io/concepts/blocks/#prefect-built-in-blocks) to store and access sensitive information.
```
Comment (jakekaplan) [inline docs/concepts/variables.md:17]:
```suggestion
    Using variables to store sensitive information, such as credentials, is not recommended. Instead, use [Secret blocks](https://docs.prefect.io/concepts/blocks/#prefect-built-in-blocks) to store and access sensitive information.
```

File: docs/concepts/variables.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+You can list, inspect, and delete variables via the command line interface with the `prefect variable ls`, `prefect variable inspect <name>`, and `prefect variable delete <name>` commands, respectively.
+
+## Accessing variables
+
+In addition to the UI and API, variables can be referenced in code and in certain Prefect configuration files.
+
+### In Python code
+
+You can access any variable via the Python SDK via the `.get()` method. If you attempt to reference a varible that does not exist, the method will return `None`.
```
Comment (jakekaplan) [inline docs/concepts/variables.md:53]:
```suggestion
You can access any variable via the Python SDK via the `.get()` method. If you attempt to reference a variable that does not exist, the method will return `None`.
```

**Review by jakekaplan — APPROVED** [review]

**Comment by billpalombi** [comment]
Looks like my bullets aren't rendering correctly? I'm not sure why.

**Comment by jakekaplan** [comment]
> Looks like my bullets aren't rendering correctly? I'm not sure why.

I think you need a new line between the text and bullets
```
Things:

- thing 1
- thing 2
```

**Comment by billpalombi** [comment]
That did it, thanks Jake!

File: docs/concepts/variables.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+description: Prefect variables are dynamic-named, mutable string values, much like environment variables.
+tags:
+    - variables
+    - blocks
+---
+
+# Variables
+
+Variables enable you to store and reuse non-sensitive bits of data, often configuration information. Variables are dynamic-named, mutable string values, much like environment variables. Variables are scoped to a Prefect Server instance or a single workspace in Prefect Cloud.
```
Comment (cicdw) [inline docs/concepts/variables.md:10]: "dynamic-named" confuses me as a term - do you mean "named, mutable string values much like environment variables"?

File: docs/concepts/variables.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+Variables can be created or modified at any time, but are intended for values with infrequent writes and frequent reads. Variable values may be cached for quicker retrieval.
+
+While variable values are most commonly loaded during flow runtime, they can be loaded in other contexts, at any time, such that they can be used to pass configuration information to Prefect configuration files, such as project steps.
+
+!!! warning "Variables are not Encrypted"
+    Using variables to store sensitive information, such as credentials, is not recommended. Instead, use [Secret blocks](https://docs.prefect.io/concepts/blocks/#prefect-built-in-blocks) to store and access sensitive information.
+
+## Managing variables
+
+You can create, read, edit and delete variables via the Prefect UI, API, and CLI. Names must:
```
Comment (cicdw) [inline docs/concepts/variables.md:21]: could be worth saying "much like environment variable naming conventions..." to emphasize the connection?

**Review by cicdw — COMMENTED** [review]
minor comments, LGTM otherwise!

File: docs/concepts/variables.md (hunk 3 above)
Comment (billpalombi) [reply docs/concepts/variables.md:10]: Yeah thats better

File: docs/concepts/variables.md (hunk 4 above)
Comment (billpalombi) [reply docs/concepts/variables.md:21]: Yup I like that

**Review by cicdw — APPROVED** [review]

**Pull request merged**